### DISCUSSION TITLE
Activating syntax highlighting for R and Python

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23145] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Github Community!</p>
<p>I’d like to ask, “How does one activate syntax highlighting for R?” I followed the instruction, created a triple backtick for Markdown, and added “r” or “R”. However, neither shows wide variety of colors as the Ruby example in the documentation does.</p>
<p>Here’s an example to show that both “r” and “R” didn’t make any difference.<br>
<img src="https://user-images.githubusercontent.com/60270936/181075319-6d833eba-afc6-4aa5-bcbf-0c2d26824886.png" alt="sample1" data-base62-sha1="yfqqeH7O33hcCTeBupmKTaGHlLj" width="318" height="34"><br>
<img src="https://user-images.githubusercontent.com/60270936/181075321-e3031ede-bd13-418f-a9ca-342849db4db3.png" alt="sample2" data-base62-sha1="jtxCHNTwEvu7e50xW0lpm9dzomf" width="320" height="44"><br>
<img src="https://user-images.githubusercontent.com/60270936/181075322-83f7280f-4bfd-437d-81d5-86bc1fc867d2.png" alt="sample3" data-base62-sha1="v9h93EGQJhmy49zH9sw2anRXiXn" width="320" height="78"><br>
<img src="https://user-images.githubusercontent.com/60270936/181075323-bc4bb92d-39e6-450d-8050-793c68acfcd1.png" alt="sample4" data-base62-sha1="k8hDTtQqN7OuJeLdpLe9xMyQT52" width="316" height="32"></p>
<p>Here are the codes.<br>
<img src="https://user-images.githubusercontent.com/60270936/181075324-9aed256a-6512-40e4-b8aa-2a04eb4834b2.png" alt="sample11" data-base62-sha1="bHpmKt9kMQBp5OYSnfnizWT3Ynp" width="320" height="48"><br>
<img src="https://user-images.githubusercontent.com/60270936/181075325-3718685f-d9db-4548-8395-fa0eb1b35d01.png" alt="sample22" data-base62-sha1="9OWuMg3TIZ7tQaBRjTAEOzUGCXf" width="353" height="68"><br>
<img src="https://user-images.githubusercontent.com/60270936/181075326-a93bdaa1-fbc4-4eb3-a10e-f58e6d2cbace.png" alt="sample33" data-base62-sha1="zTJ0eUeyck4vy1HoRFxs6NkGxYP" width="360" height="102"></p>
<p>Logically, it should be “r” but I’m not so sure. Had I activated R syntax highlighting incorrectly?</p>
<p>Moving forward, does ```python activates syntax highlighting for Python? I’m not sure as well. I’ve tried to look for examples of activating syntax highlighting for R and Python to find none.</p>
<p>A little background is that I didn’t mess with the _config.yml as I just started web building with Github Page, so I built my site using the simplest way possible. I need to get this website done ASAP so I started coding it with only basic HTML in mind.</p>
<p>Thank you for your assistance. Stay safe!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>You can try your code in the <a href="http://rouge.jneen.net/v3.26.0/r/">Rouge sample highlighter</a> to see what the expected result should be. Rouge is the syntax highlighter used on Pages.</p>
